src/routes/transactions/mappers/transfers/transfer-info.mapper.ts

```diff
@@ -52,24 +52,29 @@ export class TransferInfoMapper {
   ): Promise<Transfer> {
     if (isERC20Transfer(domainTransfer)) {
       const { tokenAddress, value } = domainTransfer;
-      const token = await this.getToken(chainId, tokenAddress);
+      const token: Token | null = await this.getToken(
+        chainId,
+        tokenAddress,
+      ).catch(() => null);
```

Comment:
Nitpick: Just FYI, I think there are two main reasons why we couldn't get the token data: validation error, and datasource retrieval error. In the first case, we are logging the error at `ValidationErrorFactory`, but in the second case we wouldn't notice and this would just return `null` silently if I'm not wrong.

I'm OK with this behavior, but I'm interested in whether you think we could also log errors on `HttpErrorFactory` as well, or log them selectively depending on the use case (in this new `catch` block for example).

Reply:
It wouldn't be completely silent. We are logging network responses [here](https://github.com/safe-global/safe-client-gateway-nest/blob/a65d5d9d0dcd695a5d6d200b9cf2db34872c10cf/src/datasources/network/axios.network.service.ts#L78C1-L88).

However, it is true that the logging happens on the network part instead of the Datasource abstraction.

Reply:
Oh, yes, it's fine then. Of course we can think if that's the best spot for the logging but if it's not silent I'm OK with this, we can improve further 👍